configs/textdet/dbnet/dbnet_r18_fpnc_1200e_icdar2015.py

```diff
@@ -1,77 +1,32 @@
 _base_ = [
-    'dbnet_r18_fpnc.py',
+    '_base_dbnet_r18_fpnc.py',
     '../../_base_/det_datasets/icdar2015.py',
-    '../../_base_/default_runtime.py',
+    '../../_base_/textdet_default_runtime.py',
     '../../_base_/schedules/schedule_sgd_1200e.py',
 ]
 
 # dataset settings
-train_list = {{_base_.train_list}}
-test_list = {{_base_.test_list}}
-file_client_args = dict(backend='disk')
-default_hooks = dict(checkpoint=dict(type='CheckpointHook', interval=20), )
-
-train_pipeline_r18 = [
-    dict(
-        type='LoadImageFromFile',
-        file_client_args=file_client_args,
-        color_type='color_ignore_orientation'),
-    dict(
-        type='LoadOCRAnnotations',
-        with_polygon=True,
-        with_bbox=True,
-        with_label=True,
-    ),
-    dict(
-        type='TorchVisionWrapper',
-        op='ColorJitter',
-        brightness=32.0 / 255,
-        saturation=0.5),
-    dict(
-        type='ImgAugWrapper',
-        args=[['Fliplr', 0.5],
-              dict(cls='Affine', rotate=[-10, 10]), ['Resize', [0.5, 3.0]]]),
-    dict(type='RandomCrop', min_side_ratio=0.1),
-    dict(type='Resize', scale=(640, 640), keep_ratio=True),
-    dict(type='Pad', size=(640, 640)),
-    dict(
-        type='PackTextDetInputs',
-        meta_keys=('img_path', 'ori_shape', 'img_shape'))
-]
-
-test_pipeline_1333_736 = [
-    dict(
-        type='LoadImageFromFile',
-        file_client_args=file_client_args,
-        color_type='color_ignore_orientation'),
-    dict(type='Resize', scale=(1333, 736), keep_ratio=True),
-    dict(
-        type='PackTextDetInputs',
-        meta_keys=('img_path', 'ori_shape', 'img_shape', 'scale_factor',
-                   'instances'))
-]
+train_list = [_base_.ic15_det_train]
+test_list = [_base_.ic15_det_test]
 
 train_dataloader = dict(
     batch_size=16,
     num_workers=8,
-    persistent_workers=False,
+    persistent_workers=True,
     sampler=dict(type='DefaultSampler', shuffle=True),
     dataset=dict(
-        type='ConcatDataset', datasets=train_list,
-        pipeline=train_pipeline_r18))
+        type='ConcatDataset',
+        datasets=train_list,
+        pipeline=_base_.train_pipeline))
 
```

Comment:
do we need 'ConcatDataset'?

```
ic15_det_train = _base_.ic15_det_train
ic15_det_train.pipeline = train_pipeline
train_dataloader = dict(
    batch_size=16,
    num_workers=8,
    persistent_workers=True,
    sampler=dict(type='DefaultSampler', shuffle=True),
    dataset=ic15_det_train)
```

Reply:
We can leave `ConcatDataset` as a convention since it makes things eaiser

Reply:
It's a little difficult to debug and  use some features in OCRDataset like `indices`